### PR TITLE
Add check for ECLBASE being present when config contains SUMMARY

### LINKED ISSUE #3241: Running a case with SUMMARY without ECLBASE causes segfault
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior using the snake_oil case:
1. `pip install ert`
1. `cd test-data/local/snake_oil`
1. Open `snake_oil.ert` and remove `ECLBASE`
1. `ert gui snake_oil.ert`
1. Run experiment (Single test run)
1. Experience ert fail

**Expected behavior**
Would expect to get a warning when starting ert that `ECLBASE` must be specified.

### PR DESCRIPTION
**Pull request by mortalisk**
**Issue**
Resolves #3241

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Review by eivindjahren — APPROVED** [review]
Nice! Could you also create a pull request for this to the 4.2.0 branch?

File: test-data/configuration_tests/ensemble_config.ert [hunk 1]
```diff
@@ -28,3 +28,5 @@ SUMMARY WWIR:INJ
 SUMMARY WWIT:INJ
 SUMMARY WBHP:INJ
 SUMMARY ROE:1
+
+ECLBASE some/path
```
Comment (eivindjahren) [inline test-data/configuration_tests/ensemble_config.ert:32]: This should probably be `input/SPE1`.

**Review by eivindjahren — CHANGES_REQUESTED** [review]
Had a comment about the `ensemble_config.ert`

**Comment by mortalisk** [comment]
> Nice! Could you also create a pull request for this to the 4.2.0 branch?

Which 4.2.0 branch?

**Review by eivindjahren — APPROVED** [review]

**Pull request merged**